app/src/main/kotlin/com/hippo/ehviewer/EhApplication.kt

```diff
@@ -248,3 +244,15 @@ class EhApplication : Application(), SingletonImageLoader.Factory {
         val searchDatabase by lazy { roomDb<SearchDatabase>("search_database.db") }
     }
 }
+
+private fun <T : HttpClientEngineConfig> HttpClientConfig<T>.commonConfigure(redirect: Boolean = true) {
+    followRedirects = false
+    if (redirect) {
+        install(HttpRedirect) {
+            checkHttpMethod = false
```

Comment:
This ain't working.
`HttpRedirect` will copy the original request method and body.
https://github.com/ktorio/ktor/blob/eea241c18961916c101307185aa08e538c754b5a/ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpRedirect.kt#L72

Reply:
Gotcha